### ISSUE TITLE
Country (gl) and language (hl) options

### ISSUE THREAD

**Issue by dittaeva** [chrisle/seer.js-public#1]
I know how to adapt the script by prefixing "&hl=no&gl=NO" to "&num=" on line 37 in the script, but it would be easier and more usable if these were options in the script the way "&num" is.

I suppose "pws=0" is irrelevant when we're not getting the results with our browser?

**Comment by chrisle** [comment]
That's an idea.  Google has a lot of options, though.  So the question is - which ones should be included and how should they be added?

I'm thinking that having them all in the argument list might make things very messy.  For example:

```
googleScraper(keyword, optResults, optCountry, optLanguage, optPersonalizedSearch, opt.......)
```

The list could become very long.

**Comment by dittaeva** [comment]
Yeah, I know, but I don't have a solution. Of course the way you've done it with an input field would make it easier, but...
